assets/js/getting-started.js

```diff
@@ -1,19 +1,20 @@
-function setupPageControl(container, pageType) {
-    var buttons = document.querySelectorAll(container + ' .btn-' + pageType);
-    for (var i = 0; i < buttons.length; i++) {
-        buttons[i].addEventListener('click', function () {
+const setupPageControl = (container, pageType) => {
+    const buttons = document.querySelectorAll(container + ' .btn-' + pageType);
+    for (let i = 0; i < buttons.length; i++) {
+        buttons[i].addEventListener('click', () => {
             if (!this.classList.contains('active')) {
```

Comment:
Have you verified that `this` still works? IIRC, `this` does not exist in arrow functions, but it does in normal functions like how it was before. I don't really see a reason for changing all functions into arrow functions in the first place.

Reply:
Reverted the change, I guess there isn't a strong reason to really use arrow functions in most places